docs/network-flow-visibility.md

```diff
@@ -131,62 +131,64 @@ Please modify them as per your requirements.
-| egressNetworkPolicyType          | 56506         | 118      | unsigned8   |
-| egressNetworkPolicyRuleName      | 56506         | 142      | string      |
-| ingressNetworkPolicyRuleAction   | 56506         | 139      | unsigned8   |
-| egressNetworkPolicyRuleAction    | 56506         | 140      | unsigned8   |
-| tcpState                         | 56506         | 136      | string      |
-| flowType                         | 56506         | 137      | unsigned8   |
+| IPFIX Information Element        | Field ID | Type        | Description |
+|----------------------------------|----------|-------------|-------------|
+| sourcePodNamespace               | 100      | string      |             |
+| sourcePodName                    | 101      | string      |             |
+| destinationPodNamespace          | 102      | string      |             |
+| destinationPodName               | 103      | string      |             |
+| sourceNodeName                   | 104      | string      |             |
+| destinationNodeName              | 105      | string      |             |
+| destinationClusterIPv4           | 106      | ipv4Address |             |
+| destinationClusterIPv6           | 107      | ipv6Address |             |
+| destinationServicePort           | 108      | unsigned16  |             |
+| destinationServicePortName       | 109      | string      |             |
+| ingressNetworkPolicyName         | 110      | string      | Name of the ingress network policy applied on the destination Pod for this flow. |
+| ingressNetworkPolicyNamespace    | 111      | string      | Namespace of the ingress network policy applied on the destination Pod for this flow. |
+| ingressNetworkPolicyType         | 115      | unsigned8   | 1 stands for Kubernetes Network Policy. <br> 2 stands for Antrea Network Policy. <br> 3 stands for Antrea Cluster Network Policy. |
+| ingressNetworkPolicyRuleName     | 141      | string      | Name of the ingress network policy Rule applied on the destination Pod for this flow. |
+| egressNetworkPolicyName          | 112      | string      | Name of the egress network policy applied on the source Pod for this flow. |
+| egressNetworkPolicyNamespace     | 113      | string      | Namespace of the egress network policy applied on the source Pod for this flow. |
+| egressNetworkPolicyType          | 118      | unsigned8   |             |
+| egressNetworkPolicyRuleName      | 142      | string      | Name of the egress network policy rule applied on the source Pod for this flow. |
+| ingressNetworkPolicyRuleAction   | 139      | unsigned8   | 1 stands for Allow. <br> 2 stands for Drop. <br> 3 stands for Reject. |
+| egressNetworkPolicyRuleAction    | 140      | unsigned8   | Name of the egress network policy Rule applied on the source Pod for this flow. |
+| tcpState                         | 136      | string      | The states of the TCP connection. |
```

Comment:
s/The states/The state

could you list them, or add a reference to the list of valid states?